src/components/requests.jsx

```diff
@@ -6,7 +6,7 @@ import {acceptGroupRequest, rejectGroupRequest,
         acceptUserRequest, rejectUserRequest} from '../redux/action-creators'
 import {tileUserListFactory, WITH_REQUEST_HANDLES} from './tile-user-list'
 
-const TileList = tileUserListFactory({type: WITH_REQUEST_HANDLES})
+const TileList = tileUserListFactory({type: WITH_REQUEST_HANDLES, withActions: true})
```

Comment:
`withActions` looks redundant in this case, we can git this info directly from `type`. For example: `PLAIN` - without any actions, and `WITH_REQUEST_HANDLES`, `WITH_REMOVE_AND_MAKE_ADMIN_HANDLES`, `WITH_REMOVE_ADMIN_RIGHTS` – it's the types with actions